src/tests/functional/navigation_tests.ts

```diff
@@ -116,6 +116,22 @@ export class NavigationTests extends TurboDriveTestCase {
     this.assert.equal(await this.pathname, "/src/tests/fixtures/one.html")
     this.assert.equal(await this.visitAction, "restore")
   }
+
+  async "test skip link with hash-only path"() {
+    await this.clickSelector('a[href="#main"]')
+    await this.nextBeat
+
+    this.assert.equal(await this.pathname, "/src/tests/fixtures/navigation.html")
+    this.assert.equal(await this.hash, "#main")
+    this.assert.ok(await this.isScrolledToSelector("#main"))
+
+    await this.remote.pressKeys('\uE004') // TAB
```

Comment:
@sstephenson maybe its due to this line? I'm not 100% clear on how to reach into leadfoot or whatever is driving Intern to send the proper key. If this _is_ the correct character, maybe it's a timing issue?

Reply:
You can import a `keys` object from `@theintern/leadfoot` and pass `keys.TAB` to `pressKeys`.  But, I don't think the effect is any different from passing the string `'\uE004'`.

I found theintern/intern#1038, and [this comment](https://github.com/theintern/leadfoot/issues/110#issuecomment-315122224) suggests using `Element.type()` as a workaround. I'm unclear if this problem/solution is relevant for elements beyond inputs and textareas, but apparently there are at least some known issues with `pressKeys`.

Reply:
@cwgw Thank you for the suggestion! I've made that change with some level of success.

@sstephenson the suite is now passing in Chrome, but the test continues to fail in Firefox.

Reply:
I think the tests will fail in Firefox for a couple of reasons:
1. The page is not long enough to be scrollable. In Chrome this doesn't appear to be an issue, so perhaps the method of measuring the "scroll position" varies. This is fixed by https://github.com/hotwired/turbo/pull/125/commits/cfbbf1413b67999e258b9507ff7c233e9fda7b3e
2. Tabbing to links in Firefox does not work unless System Preferences -> Keyboard -> Shortcuts -> All controls is checked. (I have tested this locally and it seems to work)

Reply:
@domchristie extending the frame seems to have fixed my testing issues! Thank you for that suggestion!

Reply:
It passes locally, but fails in CI. I wonder if there's a setting we can pass to `intern.json`:

https://github.com/hotwired/turbo/blob/bf3c3b0ad56249fa6a3eec3dd59f5d84ec897513/intern.json#L15-L17